File: 12577 - Hajj-e-Akbar/Java/src/Main.java
import java.util.Scanner;

public class Main {

    public static void main(String[] args) {
       String string ;
       int i=0;
        Scanner aObj = new Scanner(System.in);
        while (aObj.hasNext()){
            i++;
            string = aObj.nextLine();
            if(string.equals("*")){
                break;
            }else if(string.equals("Hajj")){
                System.out.println("Case "+i+":"+" Hajj-e-Akbar");
            }else if(string.equals("Umrah")){
                System.out.println("Case "+i+":"+" Hajj-e-Asghar");
            }
        }

    }
}

File: 12577 - Hajj-e-Akbar/C/main.cpp
#include <iostream>
#include<stdio.h>
#include <cstring>

int main() {
    char c[20];
    int i=0;
    while (strcmp(gets(c), "*")!=0) {
        i++;
        if (strcmp(c, "Hajj")==0) {
            printf("Case %d: Hajj-e-Akbar\n",i);
        } else if (strcmp(c, "Umrah")==0) {
            printf("Case %d: Hajj-e-Asghar\n",i);
        }
    }
}